sky/utils/schemas.py

```diff
@@ -619,8 +650,11 @@ def get_config_schema():
         'required': [],
         'additionalProperties': False,
         'properties': {
+            'jobs': controller_resources_schema,
```

Comment:
should this be jobs or job? should we keep it align with the cli

Reply:
We now changed all to `jobs` now to keep CLI, API, and config aligned.

Reply:
sounds good